packages/screens/DAppStore/query/getFromFile.ts

```diff
@@ -0,0 +1,365 @@
+          selectedByDefault: true,
+          alwaysOn: true,
+        },
+        launchpad: {
+          id: "launchpad",
+          title: "Launchpad",
+          description: "Multi Network NFT Launcher",
+          icon: launchpad,
+          route: "Launchpad",
+          groupKey: "teritori-core-apps",
+          selectedByDefault: true,
+          alwaysOn: true,
+        },
+        namespace: {
+          id: "namespace",
+          title: "Name Service",
+          description: "Teritori Name Service",
+          icon: tnsService,
+          route: "TNSHome",
+          groupKey: "teritori-core-apps",
+          selectedByDefault: true,
+          alwaysOn: true,
+        },
+        wallet: {
+          id: "wallet",
+          title: "My Wallet",
+          description: "Wallet",
+          icon: wallet,
+          route: "WalletManager",
+          groupKey: "teritori-core-apps",
+          selectedByDefault: true,
+          alwaysOn: true,
+        },
+        staking: {
+          id: "staking",
+          title: "Staking",
+          description: "Staking",
+          icon: staking,
+          route: "Staking",
+          groupKey: "teritori-core-apps",
+          selectedByDefault: true,
+          alwaysOn: false,
+        },
+        governance: {
+          id: "governance",
+          title: "Governance",
+          description: "Governance",
+          icon: governance,
+          route: "Governance",
+          groupKey: "teritori-core-apps",
+          selectedByDefault: true,
+          alwaysOn: false,
+        },
+      },
+    },
+    "top-apps": {
+      id: "top-apps",
+      groupName: "Top dApps",
+      icon: checklogo,
+      active: true,
+      options: {
+        osmosis: {
+          id: "osmosis",
+          title: "Osmosis Dex",
+          description: "Advanced automated market maker (AMM)",
+          icon: osmosisSVG,
+          route: "Swap",
+          groupKey: "top-apps",
+          selectedByDefault: true,
+          alwaysOn: false,
+        },
+        "social-feed": {
+          id: "social-feed",
+          title: "Social Feed",
+          description: "Decentralized Social Feed",
+          icon: socialFeed,
+          route: "Feed",
+          groupKey: "top-apps",
+          selectedByDefault: true,
+          alwaysOn: true,
+        },
+        riotersGame: {
+          id: "riotersGame",
+          title: "Join The R!ot",
+          description: "Join The R!ot",
+          icon: riot,
+          route: "RiotGame",
+          groupKey: "top-apps",
+          selectedByDefault: true,
+          alwaysOn: true,
+        },
+        toripunks: {
+          id: "toripunks",
+          title: "Toripunks dApp",
+          icon: toripunks,
+          description: "Enter the Bar, play games. punks!",
+          route: "ToriPunks",
+          groupKey: "top-apps",
+          selectedByDefault: false,
+          alwaysOn: false,
+        },
+      },
+    },
+    "coming-soon": {
```

Comment:
I think this should not be hardcoded, also what happens if one of those is removed from remote source?